packages/pages/etc/pages.api.md

```diff
@@ -3,18 +3,56 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
-/// <reference types="react" />
+import { Command } from "commander";
+import { default as React_2 } from "react";
 
 // @public
 export type Attributes = Record<string, string>;
 
+// Warning: (ae-internal-missing-underscore) The name "ClientServerRenderTemplates" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal
+export interface ClientServerRenderTemplates {
```

Comment:
These 4 internal funcs shouldn't have docs. We shouldn't need to do the underscore thing it's suggesting. I think we just need to not export them at the top level wherever we are.

Reply:
There are not a lot of [configurable settings](https://api-extractor.com/pages/configs/api-extractor_json/#api-report-section) available for how the api-extractor creates the API Report.

Reply:
Are you saying that .md files are generated but documentation doesn't get added to doc/api? Actually I think that's ok. I think I was misremembering that md files were only for public things, but it's actually for everything and then only public things to to docs/api.